test/dlc_tests/conftest.py

```diff
@@ -58,6 +58,7 @@
     "eia",
```

Comment:
if we are going to make the lookup change, let's add comments grouping these fixtures into the three accepted conditions I mentioned (repo_name fixtures, processor fixtures, job type fixtures)